content/_code-samples/delete-account/py/delete-account.py

```diff
@@ -0,0 +1,58 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import AccountSet, SetRegularKey, AccountSetFlag
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.models.requests import AccountInfo
+
+# This code sample will blackhole an account
+# A funded account on the testnet is provided for testing purposes
+# https://xrpl.org/accounts.html#special-addresses
+
+asfDisableMaster = 1114112
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Get credentials from the Testnet Faucet
+print("Requesting account from the Testnet faucet...")
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+# Construct SetRegularKey transaction
+tx_regulary_key = SetRegularKey(
+    account=myAddr,
+    regular_key="rrrrrrrrrrrrrrrrrrrrBZbvji"
+)
+
+# Sign the transaction
+tx_regulary_key_signed = safe_sign_and_autofill_transaction(tx_regulary_key, wallet=test_wallet, client=client)
+submit_tx_regular = send_reliable_submission(client=client, transaction=tx_regulary_key_signed)
+submit_tx_regular = submit_tx_regular.result
+print(f"\n SetRegularKey tx submit result: {submit_tx_regular['meta']['TransactionResult']}")
+print(f"                     Tx content: {submit_tx_regular}")
+
+# Construct AccountSet transaction w/ asfDisableMaster flag
+# This permanently blackholes an account!
+tx_disable_master_key = AccountSet(
+    account=myAddr,
+    set_flag=AccountSetFlag.ASF_DISABLE_MASTER
+)
+
+# Sign the transaction
+tx_disable_master_key_signed = safe_sign_and_autofill_transaction(tx_disable_master_key, wallet=test_wallet, client=client)
+submit_tx_disable = send_reliable_submission(client=client, transaction=tx_disable_master_key_signed)
+submit_tx_disable = submit_tx_disable.result
+print(f"\n Disable Master Key tx submit result: {submit_tx_disable['meta']['TransactionResult']}")
+print(f"                          Tx content: {submit_tx_disable}")
+
+# Verify Account Settings
+get_acc_flag = AccountInfo(
+    account=myAddr
+)
+response = client.request(get_acc_flag)
+
+if response.result['account_data']['Flags'] & asfDisableMaster:
```

Comment:
It might be helpful to show what happens when you try to do a new transaction after it has been black holed.

Reply:
You're right, thank you :)

Reply:
xrpl-py won't allow you to send transactions with a blackholed account, it'll fail:
```
raise XRPLReliableSubmissionException(
xrpl.asyncio.transaction.reliable_submission.XRPLReliableSubmissionException: The latest ledger sequence 34856252 is greater than the last ledger sequence 34856252 in the transaction. Prelim result: tefMASTER_DISABLED
```
This may confuse new developers, so I'll just add a column displaying the account's address and seed and if they want to test it out for themselves, they could use it on a Payment transaction